bedrock/mozorg/templates/mozorg/impact-report/index.html

```diff
@@ -0,0 +1,139 @@
+{#
+ This Source Code Form is subject to the terms of the Mozilla Public
+ License, v. 2.0. If a copy of the MPL was not distributed with this
+ file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#}
+
+{% extends "base-protocol-mozilla.html" %}
+
+{% from "macros-protocol.html" import split, picto, card with context %}
+
+{% block page_title %}Social and Environmental Impact{% endblock %}
+{% block page_desc %}Mozilla supports making the internet more equitable, inclusive, and sustainable.{% endblock %}
+
+{% block page_css %}
+  {{ css_bundle('protocol-split') }}
+  {{ css_bundle('protocol-card') }}
+  {{ css_bundle('impact') }}
+{% endblock %}
+
+{% block content %}
+  <main>
+    <header>
+      <div class="c-hero-banner mzp-l-content">
+
+          <h1>
+            <span>Our social</span>
+            <span>&</span> <br>
+            <span>environmental</span> <br> <span>impact</span>
+          </h1>
+
+        </div>
+        <div class="c-subheader mzp-l-content">
+          <h2>Learn how Mozilla is making the internet more equitable, inclusive, and sustainable</h2>
+        </div>
+
+    </header>
+
+    <section class="c-report">
+      {% call split(
+        block_class='mzp-l-split-center-on-sm-md mzp-l-split-reversed c-foundation',
+        image=resp_img(
+          url='img/mozorg/impact/report-cover.jpg',
+          srcset={
+            'img/mozorg/impact/report-cover-high-res.jpg': '1.5x'
+          },
+          optional_attributes={
+            'class': 'mzp-c-split-media-asset',
+            'loading': 'lazy'
+          }
+        ),
+        media_after=False
+    ) %}
+    {% if switch('impact-report-active') %}
+        <h3>Our 2024 report</h3>
+        <p>This holistic report by the Mozilla Corporation and Mozilla Foundation provides transparency to our employees, customers, and community on the progress of our Sustainability and Diversity, Equity, Inclusion, and Belonging commitments in 2023.</p>
+          <a href="https://assets.mozilla.net/pdf/impact-report" class="mzp-c-button">View the report</a>
```

Comment:
This should be the PDF report name when we upload it to the assets repo

Reply:
It will probably need to have an extension: 
```suggestion
          <a href="https://assets.mozilla.net/pdf/impact-report.pdf" class="mzp-c-button">View the report</a>
```